numba/cuda/tests/cudapy/test_compiler.py

```diff
@@ -192,15 +193,37 @@ def test_c_abi_with_abi_name(self):
         self.assertRegex(ptx, r"\.visible\s+\.func\s+\(\.param\s+\.b32\s+"
                               r"func_retval0\)\s+_Z4funcii\(")
 
+    def test_compile_defaults_to_c_abi(self):
+        ptx, resty = compile(f_module, int32(int32, int32), device=True)
+
+        # The function name should match the Python function name, and its
+        # return value should be 32 bits
+        self.assertRegex(ptx, r"\.visible\s+\.func\s+\(\.param\s+\.b32\s+"
+                              r"func_retval0\)\s+f_module\(")
+
+    def test_compile_to_ltoir(self):
+        if runtime.get_version() < (11, 5):
+            self.skipTest("-gen-lto unavailable in this toolkit version")
+
+        ltoir, resty = compile(f_module, int32(int32, int32), device=True,
+                               output="ltoir")
+
+        # There are no tools to interpret the LTOIR output, but we can check
+        # that we appear to have obtained an LTOIR file. This magic number is
+        # not documented, but is expected to remain consistent.
+        LTOIR_MAGIC = 0x7F4E43ED
+        header = int.from_bytes(ltoir[:4], byteorder='little')
+        self.assertEqual(header, LTOIR_MAGIC)
+
```

Comment:
Should a test be added for compiling to a kernel with LTO-IR as output? Or does this not make any sense as a combination?

Reply:
For now I'd omitted it because there's no practical way to make use of such an output - we'd need to provide a way to compile the kernel with a C ABI (or more C-like ABI) and to be able to know or enforce the name of the generated kernel. I do want to do this at some future point, and I think that point would be an appropriate time to add the test.

Reply:
OK, thanks for the explanation, I considered that right now it probably didn't make sense but thought I'd just check!